pandas/tests/groupby/test_groupby.py

```diff
@@ -2012,3 +2012,11 @@ def test_groups_repr_truncates(max_seq_items, expected):
 
         result = df.groupby(np.array(df.a)).groups.__repr__()
         assert result == expected
+
+
+def test_groupby_get_by_index():
+    # GH 33439
+    df = pd.DataFrame({"A": ["S", "W", "W"], "B": [1.0, 1.0, 2.0]})
```

Comment:
can you move this to pandas/tests/groupby/aggregate/test_aggregate.py

Reply:
done.